dts/arm/st/g4/stm32g441.dtsi

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2021 Guðni Már Gilbert
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/g4/stm32g431.dtsi>
```

Comment:
this file is also empty, is DMA stuff missing ?

Reply:
No it is included from `st/g4/stm32g431.dtsi` since it is the same.

Reply:
so I'd just include 431 directly

Reply:
nevermind, it's ok, other soc seem to do that.